File: PYTHON PROGRAMMING/Test39.py
# 39 PRINT WHETHER THE GIVEN YEAR VALUE IS A LEAP YEAR OR NOT
def func(year):
    if ((year % 4 == 0 and year % 100 != 0) or year % 400 == 0):
        print(f"{year} Is Leap Year!")
    else:
        print(f"{year} Is Not Leap Year!")
func(2004)
func(2005)
func(2016)
func(2024)
func(2025)

File: JAVASCRIPT PROGRAMMING/Test39.js
"use strict";
// 39. PRINT WHETHER THE GIVEN YEAR VALUE IS A LEAP YEAR OR NOT
const func39 = (year) => {
    if ((year % 4 === 0 && year % 100 !== 0) || year % 400 === 0) {
        console.log(`${year} Is Leap Year!`);
    }
    else {
        console.log(`${year} Is Not Leap year!`);
    }
};
func39(2024);
func39(2023);
func39(2021);
func39(2020);